dev-tools/export_dashboards.py

```diff
@@ -14,13 +14,13 @@ def ExportDashboards(es, regex, kibana_index, output_directory):
     try:
         reg_exp = re.compile(regex, re.IGNORECASE)
     except:
-        print("Wrong regex {}".format(regex))
+        print(("Wrong regex {}".format(regex)))
```

Comment:
Yeah, the script didn't work as well as I hoped. It seems to just apply the same rules again and again. So it doesn't work if there is already code which is python 3 compatible. Need to check the options of the script in more detail.